docs/ledger/migrations/v4.0-migration.md

```diff
@@ -0,0 +1,325 @@
+# Migration Guide v4.0
+
+## API Changes
+
+### cosmos
+
+Regen Ledger v4.0 does not include an updated version of Cosmos SDK and therefore there are no changes to the API for Cosmos SDK modules.
+
+### regen.data
+
+Regen Ledger v4.0 introduces [regen.data.v1](https://buf.build/regen/regen-ledger/docs/main:regen.data.v1).
```

Comment:
```suggestion
Regen Ledger v4.0 introduces [regen.data.v1].

[regen.data.v1]: https://buf.build/regen/regen-ledger/docs/main:regen.data.v1
```

just a nit that might arguably improve the markdown readability. that said, we do have a bunch of links, so i'm not sure it's worth the effort. each link could precede the paragraph so they stay close the first usage. or they could be collected at the end of the doc. but again, just a nit, no strong opinion here.

Reply:
I almost forgot about reference links. Although I would prefer to use the standard link format over the reference link format in this case given no two links are the same in this document and we are listing items rather than writing paragraphs.

Reply:
I like the idea but I would prefer to use the standard link format over the reference link format in this case given no two links are the same in this document and we are listing items rather than writing paragraphs.